blocks/abp-review/components/helixReview/HelixReview.js

```diff
@@ -21,7 +22,7 @@ const HelixReview = ({
 }) => {
   const [avgRating, setAvgRating] = useState(5);
   const [totalReviews, setTotalReviews] = useState(0);
-  const onRatingSet = ({
+  const onRatingSet = async ({
```

Comment:
Not sure why `async` is required since I don't see any `await`s inside function body